From d71502e8276d62b3d2ecabd322b930f1660e6b1f Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Sat, 28 Apr 2012 16:24:17 -0300
Subject: [PATCH 03/40] =?UTF-8?q?4.0.2=20Clases:=20El=20N=C3=BAcleo=20de=20K?=
 =?UTF-8?q?roshius=20(Brujo)?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Correcciones en la quest en sí y añadido evento al finalizar la misma
---
 ...2012_04_02_101_world_kroshius_infernal_core.sql |   35 ++++++++++++++++++++
 1 files changed, 35 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_02_101_world_kroshius_infernal_core.sql

diff --git a/sql/updates/world/reanemu/2012_04_02_101_world_kroshius_infernal_core.sql b/sql/updates/world/reanemu/2012_04_02_101_world_kroshius_infernal_core.sql
new file mode 100644
index 0000000..6b00dd4
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_02_101_world_kroshius_infernal_core.sql
@@ -0,0 +1,35 @@
+/********************************************
+      Quest - Kroshiu's Infernal Core
+********************************************/
+UPDATE `creature` SET `dynamicflags`=32, `unit_flags`=unit_flags|2|256|131072 WHERE `id`=14467;
+DELETE FROM `gameobject` WHERE `id`=300049;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`) VALUES
+(NULL,300049,1,1,1,5780.34,-964.844,412.694,3.50811,0,0,0,0,25,0,1);
+
+DELETE FROM `event_scripts` WHERE `id`=8328;
+INSERT INTO `event_scripts` (`id`,`delay`,`command`,`datalong`,`datalong2`,`dataint`,`x`,`y`,`z`,`o`) VALUES
+(8328,1,10,14467,90000,0,5780.34,-964.844,412.694,3.50811);
+
+-- Extra: evento al entregar la quest
+SET @NIBY :=14469;
+SET @IMPSY :=14470;
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry` IN (@NIBY,@IMPSY);
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@NIBY AND `source_type`=0;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@NIBY*100 AND `source_type`=9;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@NIBY,0,0,0,20,0,100,0,7603,0,0,0,80,@NIBY*100,2,0,0,0,0,1,0,0,0,0,0,0,0,'Niby the Almighty - Quest rewarded - Run script'),
+(@NIBY*100,9,0,0,0,0,100,0,1000,1000,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,'Niby the Almighty - Script - Say text'),
+(@NIBY*100,9,1,0,0,0,100,0,4000,4000,0,0,11,23056,0,0,0,0,0,7,0,0,0,0,0,0,0,'Niby the Almighty - Script - Call Infernal Destroyer'),
+(@NIBY*100,9,2,0,0,0,100,0,1000,1000,0,0,1,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Niby the Almighty - Script - Say text'),
+(@NIBY*100,9,3,0,0,0,100,0,4000,4000,0,0,1,0,0,0,0,0,0,9,@IMPSY,0,30,0,0,0,0,'Niby the Almighty - Script - Say text Impsy'),
+(@NIBY*100,9,4,0,0,0,100,0,5000,5000,0,0,1,1,0,0,0,0,0,9,@IMPSY,0,30,0,0,0,0,'Niby the Almighty - Script - Say text Impsy'),
+(@NIBY*100,9,5,0,0,0,100,0,5000,5000,0,0,1,2,0,0,0,0,0,1,0,0,0,0,0,0,0,'Niby the Almighty - Script - Say text');
+
+DELETE FROM `creature_text` WHERE `entry` IN (@NIBY,@IMPSY);
+INSERT INTO `creature_text` (`entry`,`groupid`,`id`,`text`,`type`,`language`,`probability`,`emote`,`duration`,`sound`,`comment`) VALUES
+(@NIBY,0,0,"Stand back! Stand clear! The infernal will need to be given a wide berth!",12,0,100,1,5,0,'Niby the Almighty'),
+(@NIBY,1,0,"BOW DOWN TO THE ALMIGHTY! BOW DOWN BEFORE MY INFERNAL DESTRO... chicken?",12,0,100,0,5,0,'Niby the Almighty'),
+(@NIBY,2,0,"Silence, servant! Vengeance will be mine! Death to Stormwind! Death by chicken!",12,0,100,1,5,0,'Niby the Almighty'),
+(@IMPSY,0,0,"Impsy rolls on the floor laughing.",16,0,100,0,5,0,'Impsy'),
+(@IMPSY,1,0,"Niby, you're an idiot.",12,0,100,1,5,0,'Impsy');
\ No newline at end of file
-- 
1.7.8.msysgit.0

